packages/toolkit/src/components/ClonePipelineDialog.tsx

```diff
@@ -182,43 +219,157 @@ export const ClonePipelineDialog = ({
-                  <Form.Field
-                    control={form.control}
-                    name="id"
-                    render={({ field }) => {
-                      return (
-                        <Form.Item className="w-full">
-                          <Form.Label className="product-body-text-3-semibold">
-                            Pipeline Name
-                          </Form.Label>
-                          <Form.Control>
-                            <Input.Root>
-                              <Input.Core
-                                {...field}
-                                className="pl-2 !product-body-text-2-regular"
-                                type="text"
-                                placeholder="Pipeline name"
-                                required={false}
-                                value={field.value || ""}
-                              />
-                            </Input.Root>
-                          </Form.Control>
-                          <p className="text-semantic-fg-secondary product-body-text-3-regular">
-                            <span>
-                              Your pipeline URL in Instill AI will be:
-                            </span>
-                            <span className="ml-2 break-all product-body-text-3-semibold">
-                              {field.value !== "" || field.value
-                                ? `${env("NEXT_PUBLIC_CONSOLE_BASE_URL")}/${me
-                                    .data?.id}/pipelines/${field.value}`
-                                : null}
-                            </span>
-                          </p>
-                          <Form.Message />
-                        </Form.Item>
-                      );
-                    }}
-                  />
+                  <div className="space-y-2">
+                    <div className="flex flex-row gap-x-4">
+                      <Form.Field
+                        control={form.control}
+                        name="namespaceId"
+                        render={({ field }) => {
+                          return (
+                            <Form.Item className="w-full">
+                              <Form.Label className="product-body-text-3-semibold">
+                                Account Name
+                              </Form.Label>
+                              <Form.Control>
+                                <Select.Root
+                                  value={field?.value || ""}
+                                  onValueChange={(e) => {
+                                    field.onChange(e);
+                                    if (form.getValues("id")) {
+                                      form.trigger("id");
+                                    }
+                                  }}
+                                >
+                                  <Select.Trigger className="w-full pl-[14px]">
```

Comment:
@iamnamananand996 We need to pick the use owned namespace (his username) as default value. In this way the user won't have weird undefined state